docker/README.md

```diff
@@ -68,6 +68,30 @@ Just change the entry in ~/.docker/config.json like this (credStore instead of c
 }
 ```
 
+### Troubleshooting for M1 Apple Silicon users
+
+1. Current Dockerfile is not architecture-agnostic, so you get an error like
+```
+#0 3.023 E: Failed to fetch http://security.ubuntu.com/ubuntu/dists/bionic-security/main/binary-arm64/Packages  404  Not Found [IP: 185.125.190.36 80]
+#0 3.023 E: Some index files failed to download. They have been ignored, or old ones used instead.
+------
+failed to solve: executor failed running [/bin/sh -c apt-get update && apt-get install -y   curl   git   build-essential   software-properties-common   wget   zlib1g-dev   libreadline-dev   libyaml-dev   libffi-dev   libxml2-dev   libxslt1-dev   wait-for-it   imagemagick   unzip   libjemalloc-dev   libssl-dev   ca-certificates   gnupg]: exit code: 100
+```
+To solve this, we need to hack Dockerfile a bit.
+
+Steps to follow:
+- Comment out line 7 `RUN echo "deb http://security.ubuntu.com/ubuntu bionic-security main" >> /etc/apt/sources.list`
```

Comment:
I'm wondering if we can remove this line from the file. Commit 4a60d7ef0ee4ea3e6b939f27b0587f7079f778d7 introduced it but it doesn't say why it needs the repo.